### PR TITLE
Levels up Rinn (level 2)

### PR DESCRIPTION
**Pull request by runicfox**
* Adds a second level of Cleric.  Was tempted to multiclass, but I'll hold off on that until later on.
* Adds in the starting equipment for the Cleric class.  It looks like I never changed it from Sparks' stuff.
* Adds table for attacks
* Adds Spells/Day label/value for 1st level spells.

I'm not entirely sure what our actual experience is.  I know that we exceeded the 300 points needed for 2nd level, but I'll have to ask Sam on Monday for the precise number.

### CODE REVIEW

File: characters/rinn.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ Str  | Dex  | Con  | Int  | Wis  | Cha
 
 Proficiency | AC  | Initiative | Speed | Max HP | Hit Dice
 ----------: | --: | ---------: | ----: | -----: | -------:
-         2  |  13 |          3 |    30 |      9 | 1d8+1
+         2  |  17 |          3 |    30 |     16 | 2d8+2
```
Comment (ecowden) [inline characters/rinn.md]: I'm not able to get AC 17. Scale mail = 14 + Dex bonus, with a max of two, right? Is there another 1AC coming from somewhere I missed?

**Comment by ecowden** [comment]
I like the skills and attacks tables. Gonna have to duplicate those.

Besides my curiosity on the AC, looks good to me! :+1:

File: characters/rinn.md (hunk 1 above)
Comment (runicfox) [inline characters/rinn.md]: The extra AC comes from the Shield.  Awesome job seeing the potential discrepancy!

Comment (ecowden) [inline characters/rinn.md]: Ooo! I think you get 2AC for the shield, then!

p. 145

Comment (runicfox) [inline characters/rinn.md]: Ah!  You're right!  Great catch.

**Pull request merged**